src/my_project/tagging/comic_match_logic.py

```
@@ -89,7 +76,7 @@ def score_results(self, result: ComicVineIssueStruct) -> float:
         return score
 
     def filter_results(self, top_n: int = 5) -> list[tuple[ComicVineIssueStruct, int]]:
-        logging.info(f"Adam here you go:\n{self.query_results}")
+        logger.info(f"Adam here you go:\n{self.query_results}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)